features/formatters/fuubar.feature

```diff
@@ -0,0 +1,55 @@
+Feature: fuubar
+
+[fuubar](https://github.com/thekompanee/fuubar) is an instafailing formatter
+that uses a progress bar instead of a string of letters and dots as feedback.
+
+It provides visual feedback of how many specs are left to be run as well as
+a rough approximation of how long it will take.
+
+![demo](https://kompanee-public-assets.s3.amazonaws.com/readmes/fuubar-examples.gif)
+
+Installation
+--------------------------------------------------------------------------------
+
+```ruby
+gem install fuubar
+
+# or in your Gemfile
+
+gem 'fuubar'
+```
+
+Usage
+--------------------------------------------------------------------------------
+
+In order to use fuubar, you have three options.
+
+### Option 1: Invoke It Manually Via The Command Line
+
+```bash
+rspec --format Fuubar --color
+```
+
+### Option 2: Add It To Your Local `.rspec` File
+
+```text
+# .rspec
+
+--format Fuubar
+--color
+```
+
+### Option 3: Add It To Your `spec_helper.rb`
+
+```ruby
+# spec/spec_helper.rb
+
+RSpec.configure do |config|
+  config.add_formatter 'Fuubar'
+end
+```
+
+---
+
+For more information, you can [view the fuubar
+README](https://github.com/thekompanee/fuubar)
```

Comment:
I would write this as a scenario (or scenarios) demonstrating 3rd party formatters.

```suggestion
Feature: Configuring 3rd party formatters

RSpec supports a variety of 3rd party formatters thanks to our awesome community.

One such example is [`fuubar`](https://github.com/thekompanee/fuubar) a fast failing formatter which approximates your test suite as a progress bar rather than continuous output, providing visual feedback of how many specs are left to run.

  Scenario: Invoking a 3rd party formatter manually from the command line.
      Given the "fuubar" gem is installed
      Given a file named "spec/example_spec.rb" with:
      """ruby
      RSpec.describe "examples" do
        it "is an example" do
        end
        it "is another example" do
        end
      end
      """
    When I run `rspec --format Fuubar spec/example_spec.rb`
    Then the output should contain "<fuubar output>"

  Scenario: Invoking a 3rd party formatter via a .rspec file 
      Given the "fuubar" gem is installed
      Given a file named ".rspec" with:
      """
      --format Fuubar
      """
    And a file named "spec/example_spec.rb" with:
      """ruby
      RSpec.describe "examples" do
        it "is an example" do
        end
        it "is another example" do
        end
      end
      """
    When I run `rspec spec/example_spec.rb`
    Then the output should contain "<fuubar output>"

  Scenario: Invoking a 3rd party formatter via your spec_helper.rb
      Given the "fuubar" gem is installed
      Given a file named "spec/spec_helper.rb" with:
      """ruby
      RSpec.configure do |config|
        config.add_formatter 'Fuubar'
      end
      """
    And a file named "spec/example_spec.rb" with:
      """ruby
      RSpec.describe "examples" do
        it "is an example" do
        end
        it "is another example" do
        end
      end
      """
    When I run `rspec spec/example_spec.rb`
    Then the output should contain "<fuubar output>"